cmd/manager/main.go

```diff
@@ -187,15 +200,15 @@ func execute() {
 
 	// Get a config to talk to the apiserver
 	log.Info("Setting up client for manager")
-	cfg, err := config.GetConfig()
-	if err != nil {
-		log.Error(err, "unable to set up client config")
-		os.Exit(1)
-	}
+	cfg := ctrl.GetConfigOrDie()
+	// Setup Scheme for all resources
+	log.Info("Setting up scheme")
+	_ = controllerscheme.SetupScheme()
```

Comment:
We weren't ever before, and it's unchecked in the scaffolding. No harm in doing it though, IIRC as implemented it will never return an error but haven't looked at it recently.